.githooks/pre-commit

```diff
@@ -0,0 +1,17 @@
+echo "####### Chunks: Pre-commit hook #######"
+echo "[running] Static Check"
+if [[ ! `which staticcheck` ]]; then
+    echo "[FAILED] Missing staticcheck dependency - Install it from: https://staticcheck.io/docs/getting-started/"
+    exit 1
+else
+    staticcheck ./
+fi
+echo "[done] Static Check"
+echo "[done] Static Check"
+echo "[running] Go fmt"
+go fmt
```

Comment:
```suggestion
go fmt ./...
```

The current form only formats files in the current package. The `./...` make sure that subpackages are also formatted.

More https://stackoverflow.com/a/55319554